tests/e2e/test_testing.py

```diff
@@ -42,5 +42,5 @@ def test_expect_revert(protostar, copy_fixture):
         "test_expect_revert.cairo::test_error_was_not_raised_before_stopping_expect_revert_fail_expected"
         in result
     )
-    assert "name: RANDOM_ERROR_NAME, message:" in result
+    assert "[error_type] RANDOM_ERROR_NAME" in result
```

Comment:
I think we should have integration tests to check the expected revert and expected events error messages specifically. Logic there is still complicated. You don't have to do this in this PR but please create a task for it. Would be nice to have cases checking if proper information is provided for:
- error mismatch
- error missing
- events mismatch

Reply:
This cheatcode can't throw a mismatch error, but I can make it print all events from Starknet State instead.

Reply:
However, it's pointless because it's not possible to convert a selector to name.

Reply:
https://github.com/software-mansion/protostar/issues/227